src/game/AI/ScriptDevAI/scripts/outland/tempest_keep/arcatraz/arcatraz.cpp

```diff
@@ -90,30 +90,20 @@ static const DialogueEntry aArcatrazDialogue[] =
 };
 
 instance_arcatraz::instance_arcatraz(Map* map) : ScriptedInstance(map), DialogueHelper(aArcatrazDialogue),
-    m_resetDelayTimer(0),
-    m_entranceEventTimer(0),
-    m_killedWarders(0),
-    m_killedDefenders(0)
+    m_resetDelayTimer(0)
 {
     Initialize();
 }
 
 void instance_arcatraz::Initialize()
 {
+    instance->GetVariableManager().SetVariable(WORLD_STATE_PROTEAN_HORROR, 1);
+    instance->GetVariableManager().SetVariable(WORLD_STATE_PROTEAN_NIGHTMARE, 1);
+    m_WardenGroup = false;
```

Comment:
you prolly wanna do m_wardenGroup(false) above in the constructor instead

Reply:
done